src/audit.multiple.md

```diff
@@ -0,0 +1,88 @@
+## Multiple Roles or Resources [/api/audit/events{?role%5B%5D,resource%5B%5D,event_action,kind,limit,till,since}]
+
+### Fetch audit events matching any subset of roles or resources [GET]
+
+Fetch audit events matching any of a set of roles and/or resources the calling identity has `read` privilege on.
```

Comment:
Does that mean that all the roles and resources are OR-ed together? Can you describe the use case of why this is useful? While I am reading the doc, it would be helpful to describe the scenario in which I would use this, rather than the routes that filter on a single role or resource.